symbols_db/config.py

```diff
@@ -12,17 +13,20 @@
 DEBUG_MODE = True
 # constants
 CWD = Path.cwd()
-WRAPDB_LOCATION = CWD / "temp" / "wrapdb"
-VCPKG_LOCATION = CWD / "temp" / "vcpkg"
+WRAPDB_LOCATION = CWD/ ".." / "temp" / "wrapdb"
```

Comment:
Maybe we tweak these paths and use either a new temp directory or environment variable over time.